app/assets/javascripts/current_deploys.js

```diff
@@ -0,0 +1,31 @@
+$(function() {
+  var $badge = $('.current-deploys .badge');
+  $error     = $("#error_message");
+
+  var updateBadgeDisplay = function(number) {
+    if(number != undefined) { $badge.text(number); }
```

Comment:
would love to see spaces are used consistently. `if (...) { ... }`
note there's a missing space after `if`